app/assets/stylesheets/application.scss

```diff
@@ -16,3 +16,24 @@
 @import 'bootstrap';
 
 
+// Header styling
+
+.navbarStyle {
+    margin-bottom: 30px;
+    background-color:darkseagreen;
+}
+
+.navTitle {
+    font-size: 25px;
+    color: black;
+    font-weight: lighter;
+}
+
+
+// Body styling
+
+// may need to update with class/id names as project progresses
+// h1 {
+//     margin-top: 30px;
+// }
```

Comment:
Do you want to keep this code on main? It' would look nicer if we don't push up commented code. I think it something you could circle back around too. it'll also be saved in your commits.